thaomonster/ACNH-Collections
# Feature/profile page

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR adds router so that when a user clicks on the image of a villager it will route them to the profile page. On the profile page there is also another link in the back button that will take them back to the home page. This PR also grabs the data for a specific villager and displays it to the page. All styling is done for the profile page. 

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/Components/App/App.js ` lines: 21-28;
`src/Components/ProfilePage/ProfilePage.js ` lines: 1-59;
`src/Components/ProfilePage/_ProfilePage.scss` lines: 1-65;
`src/Components/VillagerList/VillagerList.js` lines: 12-17;
`src/apiCalls.js` lines: 13-17;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` this branch
2. run `npm i`
3. run `npm start`
4. Click on a villager
5. Click the back button

### Steps To Take
>Any questions and possible steps moving forward.

None.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #4,  Closes #5  &  Closes #15
